xbmc/storage/windows/Win32StorageProvider.cpp

```diff
@@ -49,8 +52,13 @@ void CWin32StorageProvider::Initialize()
 
 void CWin32StorageProvider::GetLocalDrives(VECSOURCES &localDrives)
 {
+  using namespace KODI::PLATFORM::WINDOWS;
   CMediaSource share;
-  share.strPath = CSpecialProtocol::TranslatePath("special://home");
+  wchar_t profilePath[MAX_PATH];
+  if (SUCCEEDED(SHGetFolderPath(NULL, CSIDL_PROFILE, NULL, 0, profilePath)))
+    share.strPath = FromW(profilePath);
+  else
+    share.strPath = CSpecialProtocol::TranslatePath("special://home");
```

Comment:
This could use the env variable `USER_PROFILE` as a fallback as that maps to the correct path and then if that doesn't exist fall back to using TranslatePath